src/components/modules/banners/contributions/ContributionsBanner.tsx

```diff
@@ -59,6 +68,89 @@ const styles = {
     ctaContainer: css`
         display: flex;
         justify-content: flex-end;
+        flex-direction: column;
+    `,
+    secondaryCtaContainer: css`
+        margin-left: ${space[4]}px;
+    `,
+    reminderContainer: css`
+        position: relative;
+        margin-top: ${space[2]}px;
+
+        ${from.tablet} {
+            margin-top: 0;
+        }
+    `,
+    reminderLine: css`
+        border-top: 1px solid black;
+        position: absolute;
+        top: 0;
+        right: 0;
+        left: 0;
+
+        ${from.leftCol} {
+            left: calc(50% - 550px + 1120px * 0.14285714285714285 - 20px + 10px + 1px);
+        }
+
+        ${from.wide} {
+            left: calc(50% - 630px + 1280px * 0.1875 - 20px + 10px);
+        }
```

Comment:
I had to get the pen and paper out to figure out these ones 😂 The difficulty is that the line starts 2/3 columns in and then goes all the way to the edge of the screen, which is a bit of a nightmare! I can maybe drawing a diagram that explains these calculations?

Reply:
this is slightly terrifying! It needs some explanation